bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,149 @@
+import re
+
+from difflib import SequenceMatcher
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+
+
+class GitRevertInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    GIT_REVERT_COMMIT_RE = re.compile(
+        r'Revert\s\".+\"\n\nThis\sreverts\scommit\s([0-9a-f]{40})\.')
+
+    def check_modified_file_similarity(self, file_path, reverted_commit_sha):
```

Comment:
I think we might wanna change this function into a private function (by adding `_` prefix in front of the function name), since this function probably wouldn't be called neither by the programmer nor user.